### PR TITLE
fix(application-system): configure expiry by application for the assignEmail token

### PR DESCRIPTION
**Pull request by jeremybarbet**
## What

- Configure the assignEmail expiry by application

### CODE REVIEW

**Review by pshomov — COMMENTED** [review]
Can I get a bit of explanation what is this token for and why do we want it not to expire. Non-expiring tokens sounds like a time-unbound access.

**Comment by jeremybarbet** [comment]
Sure, when we create an application, depending on the state you end up in, it might need a review from someone else. In this case, we assign the application to someone else through email.

For example, in the parental leave, the applicant adds their employer's email address and asks for 6 months off. It then sends an email to the employer to approves these periods. However, we don't know when the employer will open this email to review the application, hence the no expiry on the token. 

With the previous code, if the employer didn't open the email on time, the token was expired and the application was in a "broken" state where the employer couldn't access and give a review on the application.

I was also thinking to regenerate a new token if we catch an expired token and resend an email to the assigned person. I'm not sure what's the standard for this kind of behavior in general.

If you see a better solution from a security standpoint, just let me know.

**Review by kksteini — APPROVED** [review]
LGTM

File: apps/application-system/api/infra/application-system-api.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { ref, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
 const postgresInfo = {
   passwordSecret: '/k8s/application-system/api/DB_PASSWORD',
 }
+
```
Comment (pshomov) [inline apps/application-system/api/infra/application-system-api.ts:6]: revert please

**Review by pshomov — APPROVED** [review]
LGTM

**Pull request merged**